frontend/public/components/namespace.jsx

```diff
@@ -112,6 +112,9 @@ const ProjectList_ = props => {
     <p>
       To learn more, visit the OpenShift <ExternalLink href={openshiftHelpBase} text="documentation" />.
     </p>
+    <p>
+      Download the command-line tools <ExternalLink href={CLI_DOWNLOAD_LINK} text="here" />.
```

Comment:
is there an appropriate place to give them a little more of a hint about the fact that they need to pick the right version?  (can we even tell them what version their cluster is here so they know what to look for?)

Reply:
> (can we even tell them what version their cluster is here so they know what to look for?)

Yeah, I'm actually not sure. The OpenShift version endpoint no longer exists, and `ClusterVersion` is not readable to normal user. Do you know?

We could try to build the right URL if we can get the version, although I'm a little nervous something won't match up and it will be a broken link.

Reply:
i'm afraid i don't.  but barring that i'd still want to prep them to know they need to pick an appropriate dir from where you're sending them.

Reply:
I'm not even sure which is the right version to download. The numbers don't match:

<img width="1156" alt="version details openshift container platform 2019-03-01 15-36-14" src="https://user-images.githubusercontent.com/1167259/53664952-120a2280-3c38-11e9-95e0-f2d640764ebc.png">

If we could reliably build the URL, the console operator could set the right link in the console config using the `ClusterVersion`.

@brenton @oourfali Any guidance here?

Reply:
I'm a little worried we may publish CI content there.  We definitely wouldn't want customers using those.  We probably need to do something better than just give them a list of directories.  There are obviously lots of hacks we could do (create a subdir and symlink in the clients we're OK with anyone running).  How much time do we have to come up with a solution?  I'd prefer the link to just work.

Reply:
The cluster version should appear in the cluster details page in UHC.
For now it is broken due to both Hive not providing it and Telemetry integration issues. But once both are resolved we should be able to show the version.

However, if those aren't aligned with the CLI versions then it won't help. And I tend to agree with @brenton that we shouldn't use those for customers. It is nice for early feedback, but not once that goes into production.